system/webrtc/tests/test_webrtcd.py

```diff
@@ -4,6 +4,7 @@
 # for aiortc and its dependencies
 import warnings
 warnings.filterwarnings("ignore", category=DeprecationWarning)
+warnings.filterwarnings("ignore", category=RuntimeWarning) # TODO: remove this when google-crc32c publish a python3.12 wheel
```

Comment:
putting this in webrtcd.py isn't enough?

Reply:
No, all files importing aiortc will get the warning